src/backend/columnar/sql/columnar--10.2-2--10.2-3.sql

```diff
@@ -10,6 +10,17 @@
 --
 -- To do that, drop stripe_first_row_number_idx and create a unique
 -- constraint with the same name to keep the code change at minimum.
+--
+-- If we have a pg_depend entry for this index, we can not drop it as
+-- the extension depends on it. Remove the pg_depend entry if it exists.
+DELETE FROM pg_depend
+WHERE classid = 'pg_am'::regclass::oid
+    AND objid IN (select oid from pg_am where amname = 'columnar')
+    AND objsubid = 0
+    AND refclassid = 'pg_class'::regclass::oid
```

Comment:
might be worth to fully classify this entry as well, guess it's `pg_catalog`, to prevent overloads, although it might be that you can't really create `pg_` prefixed tables anyway. Still safe to qualify it here.

Reply:
The function that inserted those rows did not qualify this part. I agree that it makes sense to qualify this. However, I think it is more important to have a consistency between the two scripts.

I think we should either update both [columnar_ensure_am_depends_catalog](https://github.com/citusdata/citus/blob/580ab012bf04710604bef74a774a9f8afb656130/src/backend/columnar/sql/udfs/columnar_ensure_am_depends_catalog/latest.sql) and here or none.